web/crux/assets/install-script/install-docker.sh.hbr

```diff
@@ -28,61 +28,61 @@ case $(uname) in
     ;;
   MINGW*)
     PLATFORM="WINDOWS"
-    if ! [ -z ${MSYS_NO_PATHCONV+x} ]; then
+    if [ -n "${MSYS_NO_PATHCONV+x}" ]; then
         ORIGINAL_PATHCONV_CONFIG=$MSYS_NO_PATHCONV
     fi
 
     export MSYS_NO_PATHCONV=1
     HOST_DOCKER_SOCK_PATH="${HOST_DOCKER_SOCK_PATH:-//var/run/docker.sock}"
     ;;
   *)
-    echo "Not Suppported OS"
+    echo "Not Supported OS"
     exit 1
     ;;
 esac
 
 set_environment() {
-# Set folder for persisting data
+  # Set folder for persisting data
   PERSISTENCE_FOLDER_FROM_CRUX="{{#if rootPath}}{{{rootPath}}}{{/if}}"
-  if [ -z ${PERSISTENCE_FOLDER:-} ]; then
-    if [ -z ${PERSISTENCE_FOLDER_FROM_CRUX:-} ]; then
+  if [ -z "${PERSISTENCE_FOLDER:-}" ]; then
```

Comment:
Error: creating named volume "~/dyrectorio/dagent/": running volume create option: names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*: invalid argument

We should make this into a full path in the script. Or check this in the input field in the webUI

Reply:
I'm not sure about this one because this time I defined the directory directly (in the input using web UI), but further down it assigned the ~/.dyrectorio... directory and that was working as it should. I'm fine with it leaving it as it is now, but we should allocate time for this in the near future.

Reply:
Okay, in order to handle this better, the best we could come up now is adding a hint to prefer absolute paths over relatives and environment variables.